pkg/gadgets/trace/exec/tracer/bpf/execsnoop.bpf.c

```diff
@@ -252,6 +252,13 @@ int ig_execve_x(struct syscall_trace_exit *ctx)
 
 	event->retval = ret;
 	bpf_get_current_comm(&event->comm, sizeof(event->comm));
+
+	struct task_struct *task = (struct task_struct *)bpf_get_current_task();
+	struct task_struct *parent = BPF_CORE_READ(task, real_parent);
+	if (parent != NULL)
```

Comment:
nit: perhaps these two variables can be declared at the top and here just use them?